docs/api-docs/storefront/widgets/widget-versioning-tutorial.md

```diff
@@ -0,0 +1,246 @@
+# Widget Versioning Tutorial
+
+<div class="otp" id="no-index">
+
+### On this page
+- [Create a widget template](#create-a-widget-template)
+- [Create a widget](#create-a-widget)
+- [Update the widget template](#update-the-widget-template)
+- [Create a new widget](#create-a-new-widget)
+- [Upgrade the widget](#upgrade-the-widget)
+- [Related resources](#related-resources)
+
+</div>
+
+This article documents how to use [widget versioning](https://developer.bigcommerce.com/api-docs/store-management/widgets/overview#widget-versioning) to update a widget template without updating its widgets.
+
+###  Prerequisites
+
+* A test store.
+* API `access_token` with `content modify` scope.
+* Knowledge of the [Widgets API](https://developer.bigcommerce.com/api-docs/storefront/widgets/widgets-overview).
+
+## Create a widget template
+
+To create a widget template, send a `POST` request to [`v3/content/widget-templates`](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/createwidgettemplate).
+
+```http
+POST /stores/{{store_hash}}/v3/content/widget-templates
+Host: api.bigcommerce.com
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+  "name": "Simple Text",
+  "schema": [
+    {
+      "type": "tab",
+      "label": "Content",
+      "sections": [
+        {
+          "settings": [
+            {
+              "type": "text",
+              "label": "Text Content",
+              "id": "textContent",
+              "typeMeta": {
+                "maxLength": 1000
+              }
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "template": "<h1>Hello, World!</h1>"
+}
+```
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/createwidgettemplate#requestrunner)
+
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Note
+> * Make a note of `uuid` and `current_version_uuid`. We will use them in the steps that follow.
+
+</div>
+</div>
+</div>
+
+You can view your widget template in the store's control panel using [Page Builder](https://support.bigcommerce.com/s/article/Page-Builder), BigCommerce's powerful theme customization tool. Your newly created widget template will be located in the left pane under **Custom**. 
```

Comment:
You can view your widget template in the store's control panel using [Page Builder](https://support.bigcommerce.com/s/article/Page-Builder), BigCommerce's powerful theme customization tool. You can locate your newly created widget template in the left pane under **Custom**.